WordPress/src/test/java/org/wordpress/android/viewmodel/activitylog/ActivityLogViewModelTest.kt

```diff
@@ -380,6 +389,20 @@ class ActivityLogViewModelTest {
         assertEquals(selectedItems, viewModel.showActivityTypeFilterDialog.value!!.initialSelection)
     }
 
+    @Test
+    fun onSecondaryActionClickRestoreNavigationEventIsShowRestore() {
+        viewModel.onSecondaryActionClicked(RESTORE, event)
+
+        Assertions.assertThat(navigationEvents.last().peekContent()).isInstanceOf(ShowRestore::class.java)
```

Comment:
Minor (🔍 ): Consider moving `Assertions` to imports, just like it is done for `Assert` for the `org.junit.Assert` import.